tests/test_table.py

```diff
@@ -21,18 +21,24 @@ class TestTerraNotebookUtilsTable(SuppressWarningsMixin, unittest.TestCase):
     @testmode("workspace_access")
     def test_fetch_attribute(self):
         table_name = "simple_germline_variation"
-        filter_column = "md5sum"
+        filter_column = "pfb:md5sum"
```

Comment:
Should these tests be changed to continue to test the old prefix, for example, using something like `for filter_column in ["pfb:md5sum", "md5sum"]`?

Reply:
The "pfb:" prefix is incidental to the fixture and not related to tests.